libs/types/src/models/submission/field-translation.type.ts

```diff
@@ -0,0 +1,12 @@
+export interface FieldTranslations {
+  title: true;
+  tags: true;
+  description: true;
+  rating: true;
+  contentWarning: true;
+  useThumbnail: true;
+  allowResize: true;
+  feature: true;
```

Comment:
We forced to make this interface so it will be expandable

Reply:
Could this not just be a string union? 'a' | 'b' | 'c'

Reply:
This could't be a string union because there is no way in typescript to add new values using augmented typings 

e.g. this
```ts
declare module "@postybirb/types" {
 interface { newTranslationKey: true }
}
```
in somewhere like "websites/webiste-name/field-labels.ts" will add new values to the interface

I dont like this design too, but i dont really know how to make this thing expandable for all websites except the scenario where we will loose type checking :(

there WILL be errors because of that

Reply:
the only potential way i see is to make default field translations of type 

```ts
{ title: lingui.msg, tags: lingui.msg, rating: lingui.msg, description: lingui.msg } & Record<string, lingui.msg>```
and add eslint rule that will scan all the files where is happening assignment to the formTranslations and then check all the labels